creator-onboarding/presents.md

```diff
@@ -38,71 +36,61 @@ Add relevant tags to your project. More on tags [here](https://docs.artblocks.io
+3. Once your project information has been uploaded, **please confirm in your artist DM that you're ready for mint #0**. The Art Blocks Team will look over your project shell and then give you the go-ahead to mint #0.
 4. Mint #0 must occur before your release can be scheduled.
 
-### Payout Details 
+### Payout Details
 
-Be sure that you've input information in all field forms. If you do not have an additional payee for primary or secondary sales, you may put `0x0000000000000000000000000000000000000000` in the field and 0% for the payee percentage. 
+Be sure that you've input information in all field forms. If you do not have an additional payee for primary or secondary sales, you may put `0x0000000000000000000000000000000000000000` in the field and 0% for the payee percentage.
 
-## Initiating your MinterSuite choice
+## Initiating your Minter Suite choice
 
-If you are using the Dutch auction - Exponential Price Decrease with/without settlement or Dutch auction - Linear Price Decrease, your project will be unpaused prior to your auction’s start time. Once your project page is public, your project can be unpaused under the Danger tab any time prior to the starting time of your auction. We recommend unpausing projects the morning of your release. Once unpaused, your project will be marked as “Upcoming” and the dutch auction will automatically begin at your start time, leaving the beginning of the auction hands-free. 
+We have a variaty of minters available for you to choose from to best suit your project.
 
-## Scheduling
+For an overview of all minters available for artists, please see the [Minter Suite](readme/minter-suite.md) page.
 
-* Once mint #0 and the features script are in place, Art Blocks will work with you to schedule/announce your release.
-* After your project has been scheduled, you are free to announce and promote your project on your social media.
-* Refer to our Marketing 101 Guide [here](https://docs.artblocks.io/creator-docs/creator-onboarding/readme/marketing101/)
+For more detailed instructions on how to configure your selected minter, please see the [Minter section of the Project Form Fields Guide](readme/project-form-fields-guide.md#minter) .
 
-## Allowlist:
```

Comment:
allowlist details and configuring instructions moved to `minter-suite.md` and `project-form-fields-guide.md`, respectively.